package.json

```diff
@@ -26,19 +26,19 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "19.2.10",
-    "@angular/cdk": "19.2.15",
-    "@angular/cli": "~19.2.0",
-    "@angular/common": "19.2.10",
-    "@angular/compiler": "19.2.10",
-    "@angular/core": "19.2.10",
-    "@angular/forms": "19.2.10",
-    "@angular/material": "19.2.15",
-    "@angular/material-experimental": "19.2.15",
-    "@angular/material-moment-adapter": "19.2.15",
-    "@angular/platform-browser": "19.2.10",
-    "@angular/platform-browser-dynamic": "19.2.10",
-    "@angular/router": "19.2.10",
+    "@angular/animations": "19.2.17",
+    "@angular/cdk": "19.2.17",
+    "@angular/cli": "19.2.17",
```

Comment:
@angular-devkit/build-angular is bumped to ^19.2.19 while @angular-devkit/core/schematics stay at 19.2.9 and @angular/language-service at 19.2.10, with @angular/cli at 19.2.17. 

Should these be aligned to the same patch level to avoid build/test?

Reply:
Also package.json (and lockfile) now report 4.6.0, we should update this since now 4.7.0